pkg/virtctl/version/version.go

```diff
@@ -15,42 +14,46 @@ type version struct {
 	clientOnly bool
 }
 
+// VersionCommand creates the version command for virtctl.
 func VersionCommand() *cobra.Command {
 	v := &version{}
 	cmd := &cobra.Command{
 		Use:     "version",
 		Short:   "Print the client and server version information.",
 		Example: usage(),
-		Args:    cobra.ExactArgs(0),
+		Args:    cobra.NoArgs,
 		RunE:    v.Run,
 	}
 	cmd.Flags().BoolVarP(&v.clientOnly, "client", "c", v.clientOnly, "Client version only (no server required).")
 	cmd.SetUsageTemplate(templates.UsageTemplate())
+
 	return cmd
 }
 
+// usage returns an example usage message for the version command.
 func usage() string {
-	usage := "  # Print the client and server versions for the current context:\n"
-	usage += "  {{ProgramName}} version"
-	return usage
+	return `  # Print the client and server versions for the current context:
+  {{ProgramName}} version`
 }
 
+// Run executes the version command, printing client and server versions.
 func (v *version) Run(cmd *cobra.Command, _ []string) error {
-	cmd.Printf("Client Version: %s\n", fmt.Sprintf("%#v", client_version.Get()))
+	cmd.Printf("Client Version: %#v\n", client_version.Get())
 
-	if !v.clientOnly {
-		virCli, _, _, err := clientconfig.ClientAndNamespaceFromContext(cmd.Context())
-		if err != nil {
-			return err
-		}
+	if v.clientOnly {
+		return nil
+	}
 
-		serverInfo, err := virCli.ServerVersion().Get()
-		if err != nil {
-			return err
-		}
+	virCli, _, _, err := clientconfig.ClientAndNamespaceFromContext(cmd.Context())
```

Comment:
```suggestion
	virtClient, _, _, err := clientconfig.ClientAndNamespaceFromContext(cmd.Context())
```

nit: Usually we call the client `virtClient`